### PR TITLE
[13.0][MIG] fleet

### PR DESCRIPTION
**Pull request by Yasaie**
Migration of `fleet`
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

File: addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---Models in module 'fleet'---
+---Fields in module 'fleet'---
+fleet        / fleet.vehicle            / future_driver_id (many2one)   : NEW relation: res.partner
+fleet        / fleet.vehicle            / net_car_value (float)         : NEW
+fleet        / fleet.vehicle            / next_assignation_date (date)  : NEW
+fleet        / fleet.vehicle            / plan_to_change_car (boolean)  : NEW isrelated: related, stored
+fleet        / fleet.vehicle.cost       / company_id (many2one)         : NEW relation: res.company, hasdefault
```
Comment (pedrobaeza) [inline addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt:9]: You have to fill this field with the vehicle's company, not the default.

File: addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+fleet        / fleet.vehicle.model      / manager_id (many2one)         : NEW relation: res.users, hasdefault
```
Comment (pedrobaeza) [inline addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt:12]: Filling it with uid=1 doesn't make sense. Better to empty it (or pre-create the column for avoiding the default filling)

File: addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+NEW fleet.vehicle.model.brand: fleet.brand_corre_la_licorne
+NEW fleet.vehicle.model.brand: fleet.brand_land_rover
+NEW fleet.vehicle.model.brand: fleet.brand_tesla_motors
+DEL fleet.vehicle.model.brand: fleet.brand_corre la licorne
+DEL fleet.vehicle.model.brand: fleet.brand_land rover
+DEL fleet.vehicle.model.brand: fleet.brand_tesla motors
+NEW ir.actions.act_window: fleet.fleet_config_settings_action [renamed from l10n_be_hr_payroll_fleet module]
+NEW ir.ui.menu: fleet.fleet_config_settings_menu [renamed from l10n_be_hr_payroll_fleet module]
+# DONE: Pre-migration (renamed)
```
Comment (pedrobaeza) [inline addons/fleet/migrations/13.0.0.1/openupgrade_analysis_work.txt:79]: It's better to reorder to couple DEL with NEW record for better visibility

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by Yasaie** [comment]
Thanks @pedrobaeza. Changes made.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**